### ISSUE TITLE
WCAG 2.2 本体 : 用語「サイズの大きな (テキスト) (large scale (text))」の注記 3

### ISSUE THREAD

**Issue by caztcha** [waic/wcag22#2192]
原文
> The actual size of the character that a user sees is dependent both on the author-defined size and the user's display or user agent settings. For many mainstream body text fonts, 14 and 18 point is roughly equivalent to 1.2 and 1.5 em or to 120% or 150% of the default size for body text (assuming that the body font is 100%), but authors would need to check this for the particular fonts in use. When fonts are defined in relative units, the actual point size is calculated by the user agent for display. The point size should be obtained from the user agent, or calculated based on font metrics as the user agent does, when evaluating this success criterion. Users who have low vision would be responsible for choosing appropriate settings.

現行訳
> 利用者が目にする文字の実際のサイズは、コンテンツ制作者が定義したサイズと、利用者のディスプレイ又はユーザエージェントの設定との両方に依存している。多くの主流となっている本文テキストのフォントでは、14 ポイント及び 18 ポイントは、1.2 em 及び 1.5 em、又は、(本文フォントが 100％であると仮定して) 本文テキストのデフォルトサイズの 120％及び 150％にほぼ相当する。しかし、コンテンツ制作者は使用する特定のフォントについてこれを確認する必要がある。フォントが相対単位で定義されている場合、実際に表示されるポイントのサイズは、ユーザエージェントによって算出される。この達成基準を評価するとき、ポイントのサイズは、ユーザエージェントから取得するか、又はそのユーザエージェントが行うようにフォントメトリックに基づいて算出されるべきである。ロービジョンの利用者は、利用者自身で適切な設定を選択する責任がある。

修正案
> 利用者が目にする文字の実際のサイズは、コンテンツ制作者が定義したサイズと、利用者のディスプレイ又はユーザエージェントの設定との両方に依存している。多くの主流となっている本文テキストのフォントでは、14 ポイント及び 18 ポイントは、1.2 em 及び 1.5 em、又は、(本文フォントが 100％であると仮定して) 本文テキストのデフォルトサイズの 120％及び 150％にほぼ相当する。しかし、コンテンツ制作者は使用する特定のフォントについてこれを確認する必要があるだろう。フォントが相対単位で定義されている場合、実際に表示されるポイントのサイズは、ユーザエージェントによって算出される。この達成基準を評価するとき、ポイントのサイズは、ユーザエージェントから取得するか、又はそのユーザエージェントが行うようにフォントメトリックに基づいて算出することになるだろう。ロービジョンの利用者は、利用者自身で適切な設定を選択する責任があるだろう。
